examples/tutorials/nuxt.md

```diff
@@ -0,0 +1,423 @@
+---
+title: "Build a Nuxt app with Deno"
+url: /examples/nuxt_tutorial/
+---
+
+[Nuxt](https://nuxt.com/) is a framework that provides an intuitive way to
+create full-stack applications based on [Vue](https://vuejs.org/). It offers
+file-based routing, server-side rendering capabilities, and automatic code
```

Comment:
One of the benefits of Nuxt is how you can mix lots of different rendering models. So I suggest this small edit here:

```suggestion
file-based routing, a variety of rendering options, and automatic code
```